design-documents/graph-ql/coverage/b2b/negotiableQuotes.graphqls

```diff
@@ -71,6 +71,10 @@ type Mutation {
         input: SetNegotiableQuotePaymentMethodInput!
     ): SetNegotiableQuotePaymentMethodOutput @doc(description: "Set the payment method on the negotiable quote")
 
+    placeNegotiableQuoteOrder(
+        input: PlaceNegotiableQuoteOrderInput
+    ): PlaceOrderOutput @doc(description: "Place an order using the negotiable quote")
```

Comment:
This should have a NQ-specific output class as well.

Reply:
Schema has been updated